thriftbp/server_middlewares.go

```diff
@@ -358,15 +355,21 @@ func tHeaderProtocol2String(proto thrift.THeaderProtocolID) string {
 	}
 }
 
-// RecoverPanic is an alias of RecoverPanik.
+// RecoverPanic recovers from panics raised in the TProccessorFunction chain,
+// logs them, and records a metric indicating that the endpoint recovered from a
+// panic.
+//
+// Deprecated: This is always added as the last server middleware (with a
+// different name) to ensure that other server middlewares get the correct error
+// if panic happened.
 func RecoverPanic(name string, next thrift.TProcessorFunction) thrift.TProcessorFunction {
-	return RecoverPanik(name, next)
+	return recoverPanik(name, next)
 }
 
-// RecoverPanik recovers from panics raised in the TProccessorFunction chain,
+// recoverPanik recovers from panics raised in the TProccessorFunction chain,
```

Comment:
was this part of a tagged release?

Reply:
it's tagged in beta, not in the last non-beta release (v0.9.11)